Makefile

```diff
@@ -48,8 +48,10 @@ generate: build-folder submodules
 		./llvm-project/llvm; \
 	fi
 
-test: generate 
-	cmake --build $(BUILD_DIR) --target TRadsan-$(ARCH)-NoInstTest TRadsan-$(ARCH)-Test -j$(NPROCS)
+test-targets: generate
+	cmake --build $(BUILD_DIR) --target TRadsan-$(ARCH)-Test TRadsan-$(ARCH)-NoInstTest -j$(NPROCS)
+
+test: test-targets
```

Comment:
Separated out building the test targets and running the test targets. Technically two different actions